app/scripts/views/sign_up.js

```diff
@@ -269,7 +319,7 @@ define(function (require, exports, module) {
       this.navigate('cannot_create_account');
     },
 
-    _initAccount: function () {
+    _initAccount: function (action) {
```

Comment:
I was digging around in here the other day thinking `_initAccount` really doesn't describe what this function does. Any suggestions?